### ISSUE TITLE
Error in TypeScript type definition file

### ISSUE THREAD

**Issue by PacRu** [mihaifm/linq#77]
I'm trying to use createOrderedEnumerable method in TypeScript, but got a TypeScript compilation error when a "null" value was passed for the "comparer" argument.

In linq.d.ts this argument for createOrderedEnumerable method is declared as required, but in linq.js it is seems to be optional:
```
    var OrderedEnumerable = function (source, keySelector, comparer, descending, parent) {
        ...
        if (comparer)
            this.comparer = Utils.createLambda(comparer);
    };
```

**Comment by mihaifm** [comment]
I can make it optional  in the typescript definition. Just curious, what's the use case for using `createOrderedEnumerable` directly?

**Comment by mihaifm** [comment]
Resolved: b0169ed92c93894856b553e8777acc0f0c37af68

**Comment by PacRu** [comment]
thanks a lot! but you are right - I don't need a direct call of createOrderedEnumerable method :)   just after your question I took a look at my code one more time and (suddenly :)) realized that, first of all, I mistakenly chain multiple createOrderedEnumerable calls instead of orderBy/thenBy (C# prevents such situations), and, secondly, I decided to rewrite my code to more clear orderBy/thenBy calls.

The initial idea with multiple chained createOrderedEnumerable calls was in sorting data by multiple fields from UI.
